### PR TITLE
[15.0][ADD] product_attribute_description

### PR DESCRIPTION
**Pull request by dsolanki-initos**
Depends on https://github.com/OCA/website/pull/907

### CODE REVIEW

**Review by fshah-initos — APPROVED** [review]
Code review and functional tests.
Looks good to me.

**Comment by rousseldenis** [comment]
@dsolanki-initos Is it a real migration or a new module ?

**Comment by jans23** [comment]
Answering on behalf: That's a new module.

File: product_attribute_description/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "author": "ERP Harbor Consulting Services,"
+    "Nitrokey GmbH,"
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "summary": """
+    Product Attribute Description
+    This module adds the information icon next to the attribute name in the
+    website and display the related description on mouse hover.
+     """,
+    "depends": ["website_sale"],
```
Comment (rousseldenis) [inline product_attribute_description/__manifest__.py]: Why depending on this ???

File: product_attribute_description/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Product Attribute Description",
+    "version": "15.0.1.0.0",
+    "category": "Website",
+    "author": "ERP Harbor Consulting Services,"
```
Comment (jans23) [inline product_attribute_description/__manifest__.py:5]: Add Nitrokey GmbH too.

Comment (dsolanki-initos) [reply product_attribute_description/__manifest__.py:5]: Hello @jans23,

I've already added and you can check this URL for the same 

https://github.com/OCA/product-attribute/blob/014744062386d7c00950a59081fcff443adcb407/product_attribute_description/__manifest__.py#L5-L8

Regards,

Dhara Solanki

**Comment by Rad0van** [comment]
If it's a new module then the PR name is misleading. If it's aimed at e-commerce it should be in that repo. Or probably even better: split into 2 modules: one adding the description in this repo and the other one showing it in e-commerce. See for example https://github.com/OCA/brand/tree/14.0/product_brand and https://github.com/OCA/e-commerce/tree/14.0/website_sale_product_brand

**Comment by dsolanki-initos** [comment]
> If it's a new module then the PR name is misleading. If it's aimed at e-commerce it should be in that repo. Or probably even better: split into 2 modules: one adding the description in this repo and the other one showing it in e-commerce. See for example https://github.com/OCA/brand/tree/14.0/product_brand and https://github.com/OCA/e-commerce/tree/14.0/website_sale_product_brand

Okay I will split it

**Comment by dsolanki-initos** [comment]
> If it's a new module then the PR name is misleading. If it's aimed at e-commerce it should be in that repo. Or probably even better: split into 2 modules: one adding the description in this repo and the other one showing it in e-commerce. See for example https://github.com/OCA/brand/tree/14.0/product_brand and https://github.com/OCA/e-commerce/tree/14.0/website_sale_product_brand

FYI, I've split it into two modules here is another PR https://github.com/OCA/website/pull/907

**Comment by dsolanki-initos** [comment]
@CarlosRoca13 Can you review this PR?

**Comment by dsolanki-initos** [comment]
@chienandalu Can you please review this PR?

**Comment by dsolanki-initos** [comment]
@olgamarcocb Can you please review this PR?

**Review by ikapasi-initos — APPROVED** [review]
Code review and functional test.
LGTM

**Comment by dsolanki-initos** [comment]
@ernestotejeda Could you please review this PR?

**Comment by dsolanki-initos** [comment]
@JordiBForgeFlow Can we merge this PR?

**Comment by jans23** [comment]
@rousseldenis We did two internal reviews and incorporated yours and other reviewers feedback. Could you reopen this please?

**Comment by dsolanki-initos** [comment]
Created new PR: https://github.com/OCA/product-attribute/pull/1255

**Pull request closed without merging**